extensions/intellij/intellij_generator_plugin/build.gradle

```diff
@@ -5,7 +5,7 @@ plugins {
 }
 
 group 'com.rx_bloc'
-version '1.3.0'
+version '1.2.0'
```

Comment:
@DDavidPrime is there any reason we want to downgrade the Intellij plugin ?

Reply:
Nope. This was changed by mistake. Fixed now.